go/serialization.go

```diff
@@ -509,3 +512,51 @@ func SerializeSTHSignatureInput(sth SignedTreeHead) ([]byte, error) {
 		return nil, fmt.Errorf("unsupported STH version %d", sth.Version)
 	}
 }
+
+// SCTListSerializedLength determines the length of the required buffer should a SCT List need to be serialized
```

Comment:
You should either (a) specify this will always return >= 2 if error is nil, or (b) return the size of the list without the length prefix (i.e. >= 0), and add the length prefix in the calling function.

I mention this because the code below takes `size - 2`, which makes the assumption size is always >= 2.